apps/admin-ui/src/spa/application-rounds/[id]/review/ApplicationsTable.tsx

```diff
@@ -94,7 +118,15 @@ const COLS = [
     headerTKey: "Application.headings.phase",
     isSortable: true,
     key: "application_status",
-    transform: ({ statusView }: ApplicationView) => statusView,
+    transform: ({ status }: { status: ApplicationStatusChoice }) => {
+      const { t } = i18n;
```

Comment:
use a hook `useTranslation` for this, not the `i18n` module directly.

Reply:
Can't be used here, as `getColConfig` isn't inside a component.. but I refactored it so we don't have a `COLS` anymore, as discussed.